svm.thy

```diff
@@ -0,0 +1,353 @@
+theory svm
```

Comment:
please adhere to [theory documentation guidelines](https://isabelle.systems/conventions/theory_documentation.html), at least add creator, summary and main definitions/theorems 

Spell out SVM, use Upper_Snake_Case